CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## 7.0.0
+
+* `HTTPReader` no longer retries on HTTP errors and therefore
+  `retryFor` was removed from `NewHTTPReader`.
```

Comment:
It would be good to have a changelog line for the general change of retrying on more types of errors as well.

Reply:
Sorry I missed this comment, updating the changelog https://github.com/maxmind/geoipupdate/pull/280.